### DISCUSSION TITLE
Add option for source member sequence number to gutter

### DISCUSSION THREAD

**Discussion by bobcozzi** [codefori/vscode-ibmi#2273] [Ideas]
I am glad to see the addition of the line change date view option in the gutter. 
I have an issue with old code getting a Decimal Data Error and it provides the statement number, however in VS CODE for i I am unable to see the SRCSEQ column. I don't need to change it, just see its value. Perhaps add it to the config like the Date Change (SRCDAT) column?

**Comment by worksofliam** [comment]
We don't plan to add the source sequence number to the editor. We are open to PRs.

**Reply by bobcozzi** [reply]
😕

**Reply by jkyeung** [reply]
Honestly, I am not a fan of this idea. Getting the dates was difficult and awkward enough (from both a technical and UI standpoint). Adding another thing to the gutter (or perhaps a whole new gutter? I have no idea about the inner workings) strikes me as a lot of effort for dubious benefit.

This is a case where I believe it is better to fit the workflow to the tool than the other way around. The main workflow adjustment is to stop preserving the sequence numbers; let them auto-renumber on save (in SEU), so they match the line numbers shown in VS Code. I get that this doesn't help you with already-compiled members and active messages waiting on QSYSOPR. For those, I would use SEU to track down the exact statement (even if it's just to get my bearings on the same member open in VS Code).
